sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -146,7 +149,11 @@ export const studentRoutes: Array<RouteRecordRaw> = [
         path: AppRoutes.StudentApplicationDetails,
         name: StudentRoutesConst.STUDENT_APPLICATION_DETAILS,
         component: StudentApplicationDetails,
-        props: true,
+        props: (route) => {
```

Comment:
you can avoid return, if you use 
![image](https://github.com/bcgov/SIMS/assets/77353155/62e04a87-c8f2-4cb5-afe4-58ebe9701c26)